.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "misoc/cores/vexriscv/verilog"]
 	path = misoc/cores/vexriscv/verilog
 	url = https://github.com/m-labs/VexRiscv-verilog.git
+[submodule "misoc/software/llvm-project"]
```

Comment:
The older libunwind entry should probably be removed, no?

Reply:
Replaced and unified the libunwind sources in `37bd336`.